src/main/kotlin/platform/adventure/framework/AdventureLibraryKind.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.adventure.framework
+
+import com.intellij.openapi.roots.libraries.LibraryKind
+
+val ADVENTURE_LIBRARY_KIND: LibraryKind = LibraryKind.create("adventure-api")
```

Comment:
This should use our `libraryKind` top-level function like:
https://github.com/minecraft-dev/MinecraftDev/blob/f30d1d798d1439da6e708e3e8d21ccda3e0659f7/src/main/kotlin/platform/bukkit/framework/BukkitLibraryKind.kt#L16